apps/admin/backend/src/adjudication.ts

```diff
@@ -107,4 +181,10 @@ export function adjudicateWriteIn(
       initialWriteInRecord.candidateId
     );
   }
+
+  await logWriteInAdjudication({
```

Comment:
It just hit me that it's kind of weird that our logging function is `async`. Seems unnecessary to me

Reply:
Yeah...